Makefile

```diff
@@ -12,9 +12,10 @@ grunt:
 
 grunt-process-lists:
 	grunt execute:preProcessLists --browser=$(browser) --type=$(type)
+	grunt browserify:backgroundTest --browser=$(browser) --type=$(type)
 
 grunt-dev:
-	cp -r test build/$(browser)/dev/
+	cp -Lr test build/$(browser)/dev/
```

Comment:
The `L` makes sure that any symlinked files (in `img` and `data`) are resolved and copied over. I'm not sure why I needed to do this all of a sudden.